app/routes/call.tsx

```diff
@@ -20,30 +34,183 @@ export const loader: LoaderFunction = async ({request}) => {
+
   const data = useRouteData<LoaderData>()
+  const [team] = useTeam()
+  const avatar = alexProfiles[team]
+
+  const activeSlug = pathname.replace('/call/', '')
+
+  const sortedEpisodes =
+    sortOrder === 'desc' ? data.episodes : [...data.episodes].reverse()
+
+  // An effect to scroll to the episode's position when opening a direct link
+  React.useEffect(() => {
+    if (activeSlug) {
+      document.querySelector(`[data-episode="${activeSlug}"]`)?.scrollIntoView()
+    }
+    // we really only want to run this on mount. Otherwise it will hijack the
+    // users scroll while they browse the episodes
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+
   return (
-    <div>
-      <h2>Welcome to the Call Kent Podcast</h2>
-      <Link to="./record">Record your own call</Link>
-      <div className="flex">
-        <div className="w-52 overscroll-auto">
-          <ul>
-            {data.episodes.map(episode => (
-              <li key={episode.slug}>
-                <Link to={`./${episode.slug}`}>{episode.title}</Link>
-              </li>
-            ))}
-          </ul>
+    <>
+      <HeroSection
+        title="Calls with Kent C. Dodds."
+        subtitle="You call, I'll answer."
+        imageUrl={avatar.src}
+        imageAlt={avatar.alt}
+        arrowUrl="#episodes"
+        arrowLabel="Wanna see my call logs?"
+        action={
+          <ButtonLink variant="primary" to="./record">
+            Record your call
+          </ButtonLink>
+        }
+      />
+
+      <HeaderSection
+        className="mb-16"
+        title="This thing is a bit new."
+        subTitle="But here is what we got."
+      />
+
+      {/*
+        IDEA: when there will be many episodes, we could split this by year, and
+        display it with tabs like on the podcast page. [2023, 2022, 2021]
+      */}
+      <Grid className="mb-24 lg:mb-64">
+        <div className="flex flex-col col-span-full mb-6 lg:flex-row lg:justify-between lg:mb-12">
+          <H6
+            id="episodes"
+            as="h2"
+            className="flex flex-col col-span-full mb-10 lg:flex-row lg:mb-0"
+          >
+            <span>Calls with Kent C. Dodds</span>
+            &nbsp;
+            <span>{` — ${data.episodes.length} episodes`}</span>
+          </H6>
+
+          <button
+            className="text-primary inline-flex items-center text-lg font-medium"
+            onClick={() => setSortOrder(o => (o === 'asc' ? 'desc' : 'asc'))}
+          >
+            {sortOrder === 'asc' ? (
+              <>
+                Showing oldest first
+                <ChevronUpIcon className="ml-2 text-gray-400" />
+              </>
+            ) : (
+              <>
+                Showing newest first
+                <ChevronDownIcon className="ml-2 text-gray-400" />
+              </>
+            )}
+          </button>
+        </div>
+
+        <div className="col-span-full">
+          {sortedEpisodes.map((episode, idx, {length}) => {
```

Comment:
Should we try and use `@reach/accordion` for this so we don't have to worry about making sure it's accessible?

Reply:
I'm not sure. It's just a `link` that navigates to a new page. Does it make sense to use accordion for that? We use it in `/discord`, but the accordions there don't navigate. Does that make a difference?

Happy to add it. Just wanna make sure that there is value in it.

Reply:
As long as it's keyboard accessible then I think we're good.